docs/contributing/translation/sync-guide.md

```diff
@@ -1,13 +1,32 @@
 ---
 title: Keeping Translations Up-to-date
-issue: https://github.com/gatsbyjs/gatsby/issues/21250
 ---
 
-Periodically, gatsbybot will create pull requests to keep translations repos up-to-date with the original English source. Make sure to review these PRs to ensure that your translation remains accurate.
+Periodically, gatsbybot will update your translation repo to be up-to-date with the current English repo. If there is an update to a page that is already translated, gatsbybot will create a pull request listing the conflicts between translation and the new English content. Resolving these conflicts and merging these pull requests is essential to keeping your translation repo up-to-date.
 
-> Note: the bot doesn't work yet but will come soon. Until then, see the next section to learn how to manually sync your repo.
+## Resolving sync pull requests
```

Comment:
Should we add a TODO or open an issue to update that part with one real-world example (e.g. spanish/japanese repo) that shows those two PRs and how they were merged (in best practice)?